Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -38,34 +42,91 @@ describe(
       ResourceCache.clearForSpecs();
     });
 
-    it("initializes from Uint8Array", function () {
+    function zoomTo(model, zoom) {
+      zoom = defaultValue(zoom, 4.0);
+
+      var camera = scene.camera;
+      var center = Matrix4.multiplyByPoint(
+        model.modelMatrix,
+        model.boundingSphere.center,
+        new Cartesian3()
+      );
+      var r = zoom * Math.max(model.boundingSphere.radius, camera.frustum.near);
+      camera.lookAt(center, new HeadingPitchRange(0.0, 0.0, r));
+    }
+
+    function verifyRender(model) {
+      expect(model.ready).toBe(true);
+
+      expect({
+        scene: scene,
+        time: JulianDate.fromDate(new Date("January 1, 2014 12:00:00 UTC")),
+      }).toRenderAndCall(function (rgba) {
+        expect(rgba).toEqual([0, 0, 0, 255]);
+      });
+
+      expect(scene).toRender([0, 0, 0, 255]);
+      model.show = true;
+      zoomTo(model);
+
+      expect({
+        scene: scene,
+        time: JulianDate.fromDate(new Date("January 1, 2014 12:00:00 UTC")),
+      }).toRenderAndCall(function (rgba) {
+        expect(rgba).not.toEqual([0, 0, 0, 255]);
+      });
+    }
+
+    it("initializes and renders from Uint8Array", function () {
       var resource = Resource.createIfNeeded(boxTexturedGlbUrl);
       var loadPromise = resource.fetchArrayBuffer();
       return loadPromise.then(function (buffer) {
         return loadAndZoomToModelExperimental(
-          { gltf: new Uint8Array(buffer) },
+          { gltf: new Uint8Array(buffer), show: false },
```

Comment:
I was never really a fan of how `ModelSpec` sets show to false when loading the model and sets show to true in the `verifyRender` call.

Instead `verifyRender` could just:
* Check that it renders non black
* Set show to `false` and check that it renders black
* Set show back to `true`

Or maybe `verifyRender` should just do the first bullet above and a separate test would check that toggling show on/off works.

Reply:
Good suggestion, it seemed odd to me too. I chose the latter option.